cadctl/cmd/investigate/investigate.go

```diff
@@ -280,14 +280,14 @@ func GetPDClient(webhookPayload []byte) (*pagerduty.SdkClient, error) {
 
 // checkCloudProviderSupported takes a list of supported providers and checks if the
 // cluster to investigate's provider is supported
-func checkCloudProviderSupported(ocmClient *ocm.SdkClient, externalClusterID string, supportedProviders []string) (bool, error) {
-	cloudProvider, err := ocmClient.GetCloudProviderID(externalClusterID)
```

Comment:
**nitpick (non-blocking):** We can remove this `ocmClient.GetCloudProviderID`  function as it's not unused.